.github/workflows/kind.yaml

```diff
@@ -17,12 +17,34 @@ jobs:
         run: |
           export GO111MODULE=on
           export PATH=$PATH:$(go env GOPATH)/bin
+          # Print cluster info
           kubectl cluster-info
           kubectl get pods -n kube-system
           kubectl wait --for=condition=Ready pods --all -n kube-system
-          make generate
-          make install
-          timeout 1m make run || retVal=$?
+
+          # TODO: Move to Makefile - Cert manager
```

Comment:
heads up: this will be added on a PR next week to support non-OCP clusters with required resources.
it is planned to be part of the `make deploy` when no TARGET is specified or TARGET=k8s